cmake/FetchLevelZero.cmake

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2024 Intel Corporation
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set(UR_LEVEL_ZERO_LOADER_LIBRARY "" CACHE FILEPATH "Path of the Level Zero Loader library")
+set(UR_LEVEL_ZERO_INCLUDE_DIR "" CACHE FILEPATH "Directory containing the Level Zero Headers")
```

Comment:
nit: `PATH` not `FILEPATH` (it's just a small change in cmake GUI)

Reply:
Done. I've done this in separate commit in case this breaks anything ;d